### ISSUE TITLE
Update the iTrans quick reference for HiTrans specs

### ISSUE THREAD

**Issue by morosebose** [v9y/giit#70]
The iTrans quick reference on the left-hand sidebar of the giitaayan page does not include the changes between iTrans and HiTrans. For example, it says that dashes should be preceded by a backslash. The file needs updating.

**Comment by v9y** [comment]
Yes, thanks. I need to work on creating a piece that explains the extensions/changes from iTrans to HiTrans.

**Comment by v9y** [comment]
Removed obsolete reference to ITRANS specs and linked to HiTrans instead.
